packages/daimo-common/src/key.ts

```diff
@@ -0,0 +1,33 @@
+import { Hex } from "viem";
+
+import { assert } from "./assert";
+
+const derPrefix = "0x3059301306072a8648ce3d020106082a8648ce3d03010703420004";
+
+export function isDERPubKey(pubKeyHex: Hex): boolean {
+  return (
+    pubKeyHex.startsWith(derPrefix) &&
+    pubKeyHex.length === derPrefix.length + 128
+  );
+}
+
+export function DERKeytoContractFriendlyKey(pubKeyHex: Hex): [Hex, Hex] {
```

Comment:
nit, `derKeyToContractFriendlyKey`

i recommend adding a unit test for `key.ts`

Reply:
done both.